src/pages/søknad/steg/formue/DinFormue.tsx

```diff
@@ -318,380 +329,379 @@ const DinFormue = (props: { forrigeUrl: string; nesteUrl: string; avbrytUrl: str
+                            <Input
+                                id={keyOf<FormData>('kontonummer')}
+                                name={keyOf<FormData>('kontonummer')}
+                                label={<FormattedMessage id="depositum.kontonummer" />}
+                                value={formik.values.kontonummer || ''}
+                                feil={formik.errors.kontonummer}
+                                onChange={formik.handleChange}
+                                autoComplete="off"
+                            />
+                        </div>
+                    )}
 
+                    {formik.values.eierBolig && (
                         <JaNeiSpørsmål
-                            id={keyOf<FormData>('skylderNoenMegPenger')}
+                            id={keyOf<FormData>('eierMerEnnEnBolig')}
                             className={sharedStyles.sporsmal}
-                            legend={<FormattedMessage id="skylderNoenMegPenger.label" />}
-                            feil={formik.errors.skylderNoenMegPenger}
-                            state={formik.values.skylderNoenMegPenger}
+                            legend={<FormattedMessage id="eiendom.eierAndreEiendommer" />}
+                            feil={formik.errors.eierMerEnnEnBolig}
+                            state={formik.values.eierMerEnnEnBolig}
                             onChange={(e) =>
                                 formik.setValues((v) => ({
                                     ...v,
-                                    skylderNoenMegPenger: e,
-                                    skylderNoenMegPengerBeløp: null,
+                                    eierMerEnnEnBolig: e,
+                                    verdiPåEiendom: null,
+                                    eiendomBrukesTil: null,
                                 }))
                             }
                         />
+                    )}
 
-                        {formik.values.skylderNoenMegPenger && (
+                    {formik.values.eierBolig && formik.values.eierMerEnnEnBolig && (
+                        <div className={sharedStyles.inputFelterDiv}>
                             <Input
-                                className={sharedStyles.marginBottom}
-                                id={keyOf<FormData>('skylderNoenMegPengerBeløp')}
-                                name={keyOf<FormData>('skylderNoenMegPengerBeløp')}
-                                bredde="S"
-                                label={<FormattedMessage id="skylderNoenMegPenger.beløp" />}
-                                value={formik.values.skylderNoenMegPengerBeløp || ''}
-                                feil={formik.errors.skylderNoenMegPengerBeløp}
+                                id={keyOf<FormData>('verdiPåEiendom')}
+                                name={keyOf<FormData>('verdiPåEiendom')}
+                                label={<FormattedMessage id="eiendom.samledeVerdi" />}
```

Comment:
samletVerdi eller samledeVerdier?

Reply:
Dem skriver formue på alt sammen på et input felt. Da blir det vel samlede verdier? flertall

Reply:
Begge deler fungerer for flertall mener jeg.